src/Directory.Build.targets

```diff
@@ -14,6 +14,10 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)graphql.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Using Include="System.Diagnostics.CodeAnalysis"/>
```

Comment:
This is non-conditional item. Why not in dirbuildprops?

Reply:
I can move it.